Chap_Terms.tex

```diff
@@ -39,6 +39,8 @@ \chapter{PMIx Terms and Conventions}
 
 \item \declaretermAlt{client}{clients} refers to a process that was registered with the \ac{PMIx} server prior to being started, and connects to that \ac{PMIx} server via \refapi{PMIx_Init} using its assigned namespace and rank with the information required to connect to that server being provided to the process at time of start of execution.
 
+\item \declaretermAlt{tool}{tools} refers to a process that may or may not have been registered with the \ac{PMIx} server prior to being started and intializes using \refapi{PMIx_tool_init}.
+
```

Comment:
We should cross-check this with the wording in the PMIx Roles section [here](https://github.com/pmix/pmix-standard/blob/dd1cc366a34adc3c83a4c07159bc4a558f56afc3/Chap_Introduction.tex#L214-L215). Maybe keep the definition here and just reference to it in the PMIx Roles section (Similar to what we do with the `client`).

Reply:
I have not done anything yet other than check that the definition Josh pointed out is compatible with what I've proposed here.  The definition Josh referenced doesn't say what determines if a process is tool, though it does mention that a process is a client based on calling the "client PMIx initialization routine".   It might be nice if that text had a mirror statement for tools that says a tools calls the "tool PMIx initialization routine".  Not sure how far to go with changing that text.  Would like to hear some other folk's thoughts on this.